YoutubeKit/Player/YTSwiftyConstants.swift

```diff
@@ -215,7 +215,10 @@ public enum VideoEmbedParameter {
      In that scenario, the origin parameter identifies the widget provider's domain, but YouTube Analytics should not identify the widget provider as the actual traffic source. Instead, YouTube Analytics uses the widget_referrer parameter value to identify the domain associated with the traffic source.
      */
     case widgetReferrer(String)
-    
+
+    /// Setting the parameter's value to `false` causes the player to not display information like the video title and uploader before the video starts playing. If the player is loading a playlist, and you explicitly set the parameter value to 1, then, upon loading, the player will also display thumbnail images for the videos in the playlist. Note that this functionality is only supported for the AS3 player.
+    case showInfo(Bool)
```

Comment:
Are you absolutely sure? The parameter is explicitly documented as deprecated:

https://developers.google.com/youtube/player_parameters#showinfo

If it were causing any kind of effects to the player, that has to be a bug. I really don't want to support any parameters that is explicitly deprecated and removed.

Reply:
Below is captured images on iPhone 5s

Without `showinfo`: 
<img width="643" alt="without showinfo" src="https://user-images.githubusercontent.com/7288035/63309634-9d2fbf80-c321-11e9-80cb-ceed320a77f1.png">

With `showinfo=0`:
<img width="644" alt="with showinfo" src="https://user-images.githubusercontent.com/7288035/63309638-a1f47380-c321-11e9-8530-823d7d5efa99.png">

These images captured with `<meta name="viewport">` after document ready.

Maybe Google will fix this issue but before that day comes, `showinfo` doesn't make youtube player worse. Please consider to temporary add it!